src/Swarm/TUI/Controller.hs

```diff
@@ -150,10 +153,13 @@ handleMainMenuEvent menu = \case
                   _ -> error "No first tutorial found!"
                 _ -> error "No first tutorial found!"
           startGame firstTutorial Nothing
+        Achievements -> uiState . uiMenu .= AchievementsMenu (BL.list AchievementList (V.fromList listAchievements) 1)
         Messages -> do
           runtimeState . eventLog . notificationsCount .= 0
           uiState . uiMenu .= MessagesMenu
-        About -> uiState . uiMenu .= AboutMenu
+        About -> do
+          uiState . uiMenu .= AboutMenu
+          attainAchievement PersistImmediate $ GlobalAchievement LookedAtAboutScreen
```

Comment:
This is the only achievement out of the four that is actually achievable at the moment, right?  Or did I miss something?

Reply:
At this moment there are two achievements implemented out of seven defined:

* "About Time!"
* "Call of the Void"

The latter demonstrates propagation of the achievement from within the GameState, out to the UI and persistence layers.